packages/compass-collection/src/stores/index.spec.ts

```diff
@@ -1,245 +1,11 @@
-    });
-
-    describe('collection appRegistry events', function () {
-      beforeEach(function () {
-        const mockCollectionModel: Partial<Collection> = {
-          document_count: 1,
-          index_count: 1,
-          index_size: 1,
-          status: 'ready',
-          avg_document_size: 2,
-          storage_size: 2,
-          free_storage_size: 0,
-        };
-
-        const mockInstance = {
-          databases: {
-            get: () => ({
-              collections: {
-                get: () => mockCollectionModel,
-              },
-            }),
-          },
-        };
-
-        sinon.replace(store, kInstance, mockInstance);
-        const mockGetStore: any = () => {};
-        sinon.replace(appRegistry, 'getStore', () => mockGetStore);
-      });
-
-      it('updates collection stats on appRegistry `select-namespace` event', function () {
-        appRegistry.emit('select-namespace', {
-          namespace: 'orange.pineapple',
-        });
-
-        const state = store.getState();
-        expect(state.namespace).to.equal('orange.pineapple');
-        expect(state.stats).to.deep.equal({
-          'orange.pineapple': {
-            avgDocumentSize: '2B',
-            avgIndexSize: '1B',
-            documentCount: '1',
-            indexCount: '1',
-            storageSize: '2B',
-            totalIndexSize: '1B',
-          },
-        });
-      });
-
-      it('updates and removes collection stats on appRegistry `select-namespace` event', function () {
-        appRegistry.emit('select-namespace', {
-          namespace: 'orange.pineapple',
-        });
-
-        appRegistry.emit('select-namespace', {
-          namespace: 'orange.not_pineapple',
-        });
-
-        const state = store.getState();
-        expect(state.namespace).to.equal('orange.not_pineapple');
-        expect(state.stats).to.deep.equal({
-          'orange.not_pineapple': {
-            avgDocumentSize: '2B',
-            avgIndexSize: '1B',
-            documentCount: '1',
-            indexCount: '1',
-            storageSize: '2B',
-            totalIndexSize: '1B',
-          },
-        });
-      });
-
-      it('updates collection stats on appRegistry `open-namespace-in-new-tab` event', function () {
-        appRegistry.emit('open-namespace-in-new-tab', {
-          namespace: 'test123.pineapple',
-        });
-
-        const state = store.getState();
-        expect(state.namespace).to.equal('test123.pineapple');
-        expect(state.stats).to.deep.equal({
-          'test123.pineapple': {
-            avgDocumentSize: '2B',
-            avgIndexSize: '1B',
-            documentCount: '1',
-            indexCount: '1',
-            storageSize: '2B',
-            totalIndexSize: '1B',
-          },
-        });
-
-        appRegistry.emit('open-namespace-in-new-tab', {
-          namespace: 'test123.pineappleNumberTwo',
-        });
+  // TODO: all old tests here were not relevant anymore, replace them with new
```

Comment:
☝️

Reply:
Added tests in https://github.com/mongodb-js/compass/pull/4858/commits/a677dbabb6c2dec1f469a7d727d898063d833b6a